evaluation/run_eval.py

```diff
@@ -265,8 +265,9 @@ def preprocess_function(examples):
         tokenized_targets = [
             tokenizer(
                 ans_choi,
-                padding=True,
-                max_length=args.target_max_length,
+                # padding is on the right here.
+                padding=False,
```

Comment:
Hum I don't think I understand. Are all answer choices the same number of tokens? Seems very dependent on the tokenizer and so I'd expect that in some cases there is actual padding.

Reply:
> Are all answer choices the same number of tokens?

No. You can have `acceptable` and `not acceptable` as answer choices. So that's why I am confused to see `padding=False`. I would think we want to pad on the right.

Reply:
Ah okay I understand better the question. Padding is done manually. https://github.com/bigscience-workshop/t-zero/blob/cf4a6781e4b8a0c5c5e45e40363a3beee0282318/t0/data_collator.py#L83-L90